Makefile

```diff
@@ -22,7 +22,7 @@ task/restore-keys: $(KEYFILE) task/hermes-image task/hermes-volume hermes.config
 	mkdir -p task && touch $@
 
 start: task/create-channel
-	docker-compose up -d
+	docker-compose up -d hermes-latest
```

Comment:
Specificy a service to run is simpler, otherwise we could a build specific file, e.g: `docker-build-compose.yml`